### PR TITLE
Update the invitation page

### PR DESCRIPTION
**Pull request by ZhovnirMaria**
## Purpose of this pull request

This pull request:
- adds links on a topic, for a better description of this topic 
- adds information on how to create an invitation from the admin

## Magento release version

- [x] 2.4.x

   Specify a patch release number, if applicable:

- [ ] 2.3.x

   Specify a patch release number, if applicable:

## Product editions

Is this update specific to a product edition?

- [ ] Magento Open Source only
- [x] Magento Commerce only

Is this update specific to an installed feature extension

- [ ] B2B extension
- [ ] Other feature set, please specify:

## Affected documentation pages

https://docs.magento.com/user-guide/marketing/invitations.html

### CODE REVIEW

File: src/marketing/invitations.md [hunk 1]
```diff
@@ -3,11 +3,40 @@ ee_only: true
...
 1. **Customer prepares invitations**: From the account dashboard, the customer prepares the list of recipients and completes the invitation. A custom message can be included, depending on the configuration.
 1. **Customer sends invitations**: When ready, the customer clicks the _Send Invitations_ button.
 1. **System manages transmission**: The system sends invitations in batches, according to the number set in the configuration.
 1. **Customer monitors response**: The customer monitors the status of each invitation from the account dashboard, as `Sent`, `Accepted`, or `Canceled`.
+
+## Create an invitation from the admin panel
+
+1. On the _Admin_ sidebar, go to **Marketing** > _Private Sales_ > **Invitations**.
+
+1. In the upper-right corner, click <span class="btn">Add Invitations</span>.
```
Comment (danidelcar) [inline src/marketing/invitations.md:19]: 1. In the upper-right corner, click **Add Invitations**.

File: src/marketing/invitations.md [hunk 2]
```diff
@@ -3,11 +3,40 @@ ee_only: true
...
+
+1. On a new screen, enter email addresses to invite new customers, add a custom message, choose a sender, and select an invitee group.
+
+    ![Invitations Information]({% link images/images-ee/create-invitation-page.png %}){: .zoom}
+    _Create Invitation_
+
+1. When complete, click <span class="btn">Save</span>.
```
Comment (danidelcar) [inline src/marketing/invitations.md:26]: 1. When complete, click **Save**.

**Review by danidelcar — APPROVED** [review]
approved with minor changes. Thanks @ZhovnirMaria !

**Comment by ZhovnirMaria** [comment]
Hi, @danidelcar 😊
Thank you! Done👌

**Comment by danidelcar** [comment]
> Hi, @danidelcar 😊
> Thank you! Done👌

Thanks @ZhovnirMaria !

**Comment by danidelcar** [comment]
@magento import code to magento-devdocs/merchdocs

**Comment by magento-engcom-team** [comment]
@danidelcar the branch with code successfully imported into`magento-devdocs/merchdocs` repository. Branch name: `imported-magento-merchdocs-1290`.

**Pull request merged**